fe/fe-core/src/main/java/org/apache/doris/statistics/MetaStatisticsTask.java

```diff
@@ -17,21 +17,85 @@
 
 package org.apache.doris.statistics;
 
+import org.apache.doris.catalog.Catalog;
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.Database;
+import org.apache.doris.catalog.OlapTable;
+import org.apache.doris.catalog.Table;
+import org.apache.doris.common.DdlException;
+
+import com.google.common.collect.Maps;
+
 import java.util.List;
+import java.util.Map;
 
-/*
-A statistics task that directly collects statistics by reading FE meta.
+/**
+ * A statistics task that directly collects statistics by reading FE meta.
+ * e.g. for fixed-length types such as Int type and Long type we get their size from metadata.
  */
```

Comment:
Please add comment:
1. The granularity of row count can only be table, and the type can only be table
2. The granularity of data size can only be table, and the type can only be table
3. The granularity of max and min size can only be table, and the type can only be column

Reply:
ok